internal/sbi/producer/smpolicy.go

```diff
@@ -319,7 +320,7 @@ func createSMPolicyProcedure(request models.SmPolicyContextData) (
 	bsfUri := consumer.SendNFInstancesBSF(pcf_context.GetSelf().NrfUri)
 	if bsfUri != "" {
 		bsfClient := util.GetNbsfClient(bsfUri)
-		_, resp, err = bsfClient.PCFBindingsCollectionApi.CreatePCFBinding(ctx, pcfBinding)
+		_, resp, err = bsfClient.PCFBindingsCollectionApi.CreatePCFBinding(context.Background(), pcfBinding)
```

Comment:
@KunLee76 @andy89923 
I think it also needs the token context.

Reply:
Thank you for all your assistance